# Objeto.msg
string label
int32 id
int32 distancia_central
int32 angulo_central
int32 tamanho_do_objeto
int32 risco
string tipo
